src/Cli/InstallCommand.php

```diff
@@ -21,6 +23,11 @@ protected function configure()
         ;
     }
 
+    public function setDockerInstaller(DockerInstaller $dockerInstaller)
```

Comment:
I am. I know it makes the constructor a bit messy. Shame we have to use extension here at all :)